src/api/models/BulkGen.ts

```diff
@@ -8,7 +8,7 @@ export type BulkGen = {
     readonly id: number;
     readonly errors: Array<BulkGenTask>;
     readonly file_name: string;
-    status?: 0 | 1 | 2 | 3 | 4 | 5;
+    readonly status: 0 | 1 | 2 | 3 | 4 | 5;
```

Comment:
why still status is shown as integer value here ? 
can you confirm @rushikeshiam